scrapers/wa/bills.py

```diff
@@ -447,16 +447,14 @@ def scrape_actions(self, bill, chamber, bill_url, prefile_year, second_year):
 
             rows = header.xpath("following-sibling::div[1]/div")
             for row in rows:
+                # skip later lines that are just links to files
                 if row.xpath("div[1]")[0].text_content().strip() != "":
```

Comment:
`if not`?

Reply:
Good catch, I didn't go looking for syntax improvements of the original code during my initial commit. I'll give it another look and push a new commit with changes.

Reply:
[New commit](https://github.com/openstates/openstates-scrapers/pull/4315/commits/c20c240503b1354d3845ffb5640f9bb9fdd31181) with change to this and other logic, syntax, and regex improvements (including following your prior guidance about pre-compiling all regular expressions).